KubeArmor/BPF/enforcer.bpf.c

```diff
@@ -13,6 +13,10 @@ char LICENSE[] SEC("license") = "Dual BSD/GPL";
 #define MAX_BUFFERS 1
 #define PATH_BUFFER 0
 
+/* In some of the distro, the generated
+ * vmlinux.h does not contain this value */
+#define PROC_PID_INIT_INO   0xEFFFFFFCU
+
```

Comment:
Wouldn't this overwrite if present?
Perhaps should be something like:
```suggestion
#ifndef  PROC_PID_INIT_INO
/* In some of the distro, the generated
 * vmlinux.h does not contain this value */
#define PROC_PID_INIT_INO   0xEFFFFFFCU
#endif

```

Reply:
In [<linux/proc_ns.h>](https://elixir.bootlin.com/linux/latest/source/include/linux/proc_ns.h#L46), `PROC_PID_INIT_INO` is defined as an `enum`. We cannot use `ifdef` or `ifndef` with enums.

Reply:
Okay! Then the question is would this overwrite? the enum definition?

Reply:
Yes. We cannot check for its existence, but we can overwrite enum constants with `#define`.

Reply:
It should be fine, unless Some future version of the kernel updates the version 👍🏽